docs/cli/platform/config/del.md

```diff
@@ -17,6 +17,7 @@
 **Whitelisted properties**
 
 - `bundleStoreProxy`
+- `binaryStoreProxy`
```

Comment:
Minor: Want to keep this in alphabetical order too? Looks like the rest of the list is? :)

Reply:
You are right :) 
Actually that's what I thought I was doing but for some reason at the time I did the change looks like my mind was using a different alphabet where `i` comes after `u` ;P
Good catch, thanks :+1: